canonical-web-and-design/tutorials.ubuntu.com
# Fix ONLYOFFICE tutorial

## Done
Fixed up some areas of the ONLYOFFICE tutorial to allow it to be released.

## QA
- Check out this feature branch
- Run the site using the command `./run serve tutorials`
- View the site locally in your web browser at: [http://0.0.0.0:8016/](http://0.0.0.0:8016/)
- Go through the How to Install ONLYOFFICE Desktop Editors on Ubuntu 18.04 tutorial
- Check it works
